azure-pipelines-e2e-integration-tests.yml

```diff
@@ -18,13 +18,13 @@ pool:
 
 steps:
 - pwsh: |
```

Comment:
Have you tried using the built-in dotnet install task? I think that would be way easier...

Here's where we use it in node.js:
https://github.com/Azure/azure-functions-nodejs-worker/blob/v3.x/azure-pipelines/test.yml#L91

Here's the docs:
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/dotnet-core-tool-installer?view=azure-devops

Reply:
Great point @ejizba . I will give this a try.